e2e/libs/backing_image/crd.py

```
@@ -0,0 +1,91 @@
+                    name=name
+                    )
+            except Exception as e:
+                logging(f"Finding backing image manager {name} failed with error {e}")
+                continue
+            
+            creation_time = backing_image_manager["metadata"]["creationTimestamp"]
+            fmt = "%Y-%m-%dT%H:%M:%SZ"
+            if datetime.strptime(creation_time, fmt) > datetime.strptime(last_creation_time, fmt):
+                return
+
+        assert False, f"Wait backing image manager {name} restart failed ..."
```

Ruff

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)